src/core/mrc_file.cpp

```diff
@@ -55,67 +57,75 @@ bool MRCFile::OpenFile(std::string wanted_filename, bool overwrite, bool wait_fo
 //	MyDebugAssertFalse(my_file->is_open(), "File Already Open: %s",wanted_filename);
 	CloseFile();
 
-	bool file_already_exists;
+	do_nothing = (wanted_filename.compare("/dev/null") == 0);
 
-	if (wait_for_file_to_exist)
+	if (do_nothing)
 	{
-		file_already_exists = DoesFileExistWithWait(wanted_filename, max_number_of_seconds_to_wait_for_file_to_exist);
+		filename = wanted_filename;
 	}
-	else
```

Comment:
Aren't you missing an else here now?

Reply:
You are right - thanks for catching this. Just pushed a fix for this.